fe/fe-core/src/main/java/com/starrocks/catalog/FunctionSet.java

```diff
@@ -589,6 +592,15 @@ public Function getFunction(Function desc, Function.CompareMode mode) {
             return null;
         }
 
+        // specially matching exactly one function by name
+        if (mode == Function.CompareMode.MATCH_NAME) {
```

Comment:
why need this logic?

Reply:
previous logic does not consider functions with various parameters and any data types, these functions just need to match the function name.